notebooks/opioid-prescription-prediction.ipynb

```diff
@@ -671,300 +754,102 @@
qhIeHU6tWLZKTk5k8eTILFy5k+fLlxbhjDoBL8W+0EEIIIYQQosyRwvI+3t7e9O3bl5CQEGJiYhg2bJhx5vFRnJ2dGTVqFAC1a9emdevWxMfHmxSWo0ePNhZ2Q4cO5ZNPPiEtLQ07Ozs2bdrEuHHjqF+/PgCenp706dOHHTt24OnpiYWFBampqVy6dImXXnqJ2rVrG88bGxuLt7c3rVq1AqBNmzYMGTKEmJgYunXr9siYe/ToYRJ/QEAAc+bMKda9EkIUjVqtRqN5/H9HNBrVfT/l0ffSSvKkDJIn5ZBcKYPkSRkK8mROUljex9HRkVatWrFhwwYOHz5McHAwd+/efexzitWqVTN5bW1tTWZmpskxB4f/LRm1srLCYDCQmZmJnZ0dly5dYvHixSxbtgzIfy5Sr9fTsmVLAIKDg/noo48ICgri9u3bdOrUiWnTpmFnZ0dKSgpdu3Y1uVatWrX47rvvHvs9Dxw4wMaNG0lISCAnJwe9Xs+dO3dK9JlMIV5UWq1VkfeV0mqtSzga8SxInpRB8qQckitlkDyJJ5HC8gE+Pj6MGTOG3r17Y29vT1JS0jO/xv3Fm729PRMnTuSVV14pdKxWqyU4OJjg4GCuXr3KtGnTeO+991i6dClOTk5cvnzZZPylS5dwdnYG8mdKHtwa5cqVK0yePJlly5bRo0cPLCwsOHToEBMmTJDCUogSkJ6eTVpa5mPHaDQqtFpr0tOz0OuVtZ3Ri0TypAySJ+WQXCmD5EkZCvJkTlJYPqBDhw5ERkZSp04d47H7i7NnsYfl/ecYOXIkERER1K1bl4YNG5Kbm8u5c+dQq9U0aNCAL7/8kkaNGlGjRg2sra2xtLSkXLn8tHl7exMaGkrXrl3x8vLihx9+YPv27bz//vsA2NnZodFouHjxonEpbmZmJgaDgSpVqhif2Vy3bt1TfIvUv30fhCj7UsnLy0Ovf1JX3vylRXq9oQhjhflInpRB8qQckitlkDwpg/mXKUthycMNeVq3bv3I94s7o1fY+PuP+fr6Ymdnx7x580hMTESj0aDT6Zg4cSIAZ8+eZdmyZdy4cQMrKyvatGnDjBkzAOjduzcZGRksWLCAlJQUXFxcmDNnDt27dwegfPnyTJkyhZCQEHJycujXrx9z585lypQpTJ8+nezsbOrVq8crr7zCf/7zn2J9r7i4ENnGopST7UZKBw+PBuYOQQghhBCixKkMz2IKTryQ0tIy5TdXpZhGo8bOzkbypACSK2WQPCmD5Ek5JFfKIHlShoI8mZP550yFEEIIIYQQQiiaFJbPWFxcHJ6enkV6FjMpKQl3d/eHGvDcb968eYSGhj7LEIUQQgghhBDimZLCsohGjBiBu7s7sbGxJsczMzNp1qwZHh4eXL58GS8vL06fPl3kZzGfNG7BggW8++67Tx23EEIIIYQQQpQ0ad5TDDqdjpiYGIYOHWo8tmvXLmrUqMH58+fNGNnzd+rUKWkKU8pJ8x7lkFwpw4uYJw+PBlhaWpo7DCGEEAoghWUxdOnShV27dhEfH0/jxo0BiI6OxsfHh4ULFwJw8uRJfH19+fXXX1Gr8yeEd+zYwebNm0lMTKRChQr06dOH2bNnA/lbj5w6dYqgoCAuX75MvXr1CAsLw83NDYBZs2ah1+tZsmQJkL9PZWhoKL/88gtOTk6MGjWK0NBQDh8+jIuLC7///jthYWGcO3eOe/fuUadOHaZOnWrsdJuUlES3bt1YvHgxkZGRhV6zKLy8wgCHZ3JfhRBClEapHDw4mSZNmpk7ECGEEAoghWUxaDQavL29iY6OpnHjxsTFxZGZmUnHjh1Nnqm8f3nrZ599xgcffMCKFSto2bIld+7c4ddffzU57+7du9m4cSO2trZMmzaN+fPn88knnzx0fb1ez9ixY2nVqhUbNmzg5s2bTJw48aHltGPHjqV58+aoVCrWrVtHUFAQBw4cwM7OrtjXfDQHwKUY44UQQgghhBBllRSWxeTt7U3fvn0JCQkhJiaGYcOGGWcmCxMVFcWYMWOMM4bW1tZ4eXmZjBk/fjxVq1YFYPDgwUyePLnQc505c4bLly+zY8cOLC0tqVatGuPGjSMwMNA4RqfTodPpjK+DgoLYvHkz8fHxdO7cudjXFEII8eJSq9VoNMpqx6DRqO77qazYXzSSK2WQPClDQZ7MSQrLYnJ0dDTOGB4+fJjg4GDu3r37yCY8iYmJ1K5d+5HnU6lUVKtWzfjaysqKO3fukJeX91DBevXqVbRaLVZWVsZj1atXNxmTnJzM0qVLOXPmDLdv30alUpGZmcn169ef6ppCCCFeXFqtldn3RXtaWq21uUMQRSS5UgbJk3gSKSyfgo+PD2PGjKF3797Y29uTlJT0yLHVq1fnzz//fCbXdXR0JD09naysLKyt8//lfvDac+bMoVKlSmzbts249LVFixZF2v5ECCGEuF96ejZpaZnmDqNYNBoVWq016elZ6PXy/77STHKlDJInZSjIkzlJYfkUOnToQGRkJHXq1DEee1Th5uvry6pVq3B3d6dly5ZkZ2fz66+/0qJFi8d+rjBNmzbF1dWVpUuXMnPmTNLT01m3bp3JmIyMDJycnLC1tSUrK4uIiAiysrJMxjybIjP1GZxDCCFE6ZVKXl4eer3SOuDmr7zR6w0KjP1FI7lSBsmTMph/1aEUlkX04FLXgmcmH/V+gaFDh6LRaFi0aBFJSUlYWVnRr18/Y2FZ1P0uIb950Nq1awkNDaVt27Y4OzszfPhwfvrpJ8qXLw/kz1jOmzePli1bUqVKFfz9/XFycipSrMURFxfyQrXcV6IXcWsEpZJcKcOLmCcPjwbmDkEIIYRCqAyyRlLRDh06xNtvv83PP//83K+dlpYpv7kqxTQaNXZ2NpInBZBcKYPkSRkkT8ohuVIGyZMyFOTJnMw/ZyqK5aeffjI+s3nx4kVWr15N//79zRuUEEIIIYQQ4oUmS2GfoVmzZqHX61myZEmJXSM1NZWpU6dy48YNtFo
```

Comment:
you'll want to remove these api keys and tokens

Reply:
Sorry, forgot about this. Should I clear out all of the output of the notebook as well then?